packages/@icon-magic/generate/src/plugins/svg-light-dark.ts

```diff
@@ -0,0 +1,172 @@
+/*
+- This plugin will create a master svg with two child svg's.
+- One dark and one light.
+- The children svgs will have a `display` attribute that can be used to set
+which of the two svg's will be visible.
+*/
+
+import {
+  Flavor,
+  GeneratePlugin,
+  Icon
+} from '@icon-magic/icon-models';
+import * as fs from 'fs-extra';
+import * as path from 'path';
+import Svgo from 'svgo';
+import { DOMImplementation, DOMParser, XMLSerializer } from 'xmldom';
+
+const serializeToString = new XMLSerializer().serializeToString;
+
+export interface SvgLightDarkOptions {
+  lightToken?: string;
+  darkToken?: string;
+}
+
+export const svgLightDark: GeneratePlugin = {
+  name: 'svg-light-dark',
+  fn: async (
+    flavor: Flavor,
+    icon: Icon,
+    params: SvgLightDarkOptions = {},
+  ): Promise<Flavor> => {
+    // Create the output directory
```

Comment:
```suggestion
    // Get the output directory
```

you're getting it not creating